client/src/app/site/motions/services/motion-csv-export.service.ts

```diff
@@ -52,8 +57,7 @@ export class MotionCsvExportService {
                 { property: 'title' },
                 {
                     label: 'recommendation',
-                    map: motion =>
-                        motion.recommendation ? this.translate.instant(motion.recommendation.recommendation_label) : ''
+                    map: motion => (motion.recommendation ? this.motionRepo.getExtendedRecommendationLabel(motion) : '')
```

Comment:
this works, but it is for the call list only. Please add your fix also for the motion list export - for extended state and recommenation label.